### ISSUE TITLE
Feature request: Show source html instead of rendering it.

### ISSUE THREAD

**Issue by Jmuccigr** [uranusjr/macdown#331]
It'd be nice if one could toggle between the code that's being generated and its output. I find myself often wanting to see what html was generated in order to find mistakes in my markdown. Copying it to the clipboard is handy, but not ideal.

**Comment by uranusjr** [comment]
One way to see the HTML is to enable the OS X built-in WebKit developer tools for MacDown:

    defaults write com.uranusjr.macdown WebKitDeveloperExtras -bool true

And select “Inspect Element” in the right-click context menu inside the preview pane.

This is the exact same inspector you find in Safari if you turn on the developer tools. It might take some time to get used to, but it is way more powerful than anything I can ever write myself.

Maybe this should be exposed in the preference menu, too. Can be useful for those who know how to use it?

**Comment by Jmuccigr** [comment]
Yes, that works for me, so consider this a request to add that as a standard pref.

Can you also make that window stay in front all the time? (Palette window?)

**Comment by uranusjr** [comment]
Not sure; I’ll look into this.

**Comment by Jmuccigr** [comment]
Thanks. (Have you had a look at the other issues I've added? I think they're bugs.)

**Comment by Jmuccigr** [comment]
Just had to search for this tip. Any progresso on adding it to the prefs? I just put in a PR ( #1005 ) for adding it to the help page.
